README.md

```diff
@@ -10,24 +10,24 @@ Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app w
 
 ### Cpu-Tracking Monitor:
 
-The Cpu-Tracking Monitor provides the possibility to add services which are to be monitored. These services as of now only comprise the database service. The services that are added need
+The Cpu-Tracking Monitor provides the possibility to add services which are to be monitored. These services as of now only comprise the Database Service. The services that are added need
 to be importing the [cpu-utilization-observer](https://github.com/ccims/cpu-utilization-observer) package which exposes an endpoint that the backend of the Cpu-Tracking Monitor scrapes for
 the cpu load. 
 
 
 ### Error-Response Monitor:
 
-The Error-Response Monitor lets one define certain request parameters in its UI for testing the database service in a environment that closely resembles its use cases. The user can adjust the type of request (GET or POST) and the expected response 
+The Error-Response Monitor lets one define certain request parameters in its UI for testing the Database Service in a environment that closely resembles its use cases. The user can adjust the type of request (GET or POST) and the expected response 
 which in the case of a GET request is the response data and in the case of a POST request is the HTTP status code. In its UI, one can also fetch all logs received by the backend of the Error-Response Monitor
-and observe specifically the logs created by work checking the database service.
+and observe specifically the logs created by work checking the Database Service.
 
 ### Monitoring Selection:
 
-The combined frontend also offers the feature to register multiple services that are to be monitored. Upon registration, the Issue Creator will only assess those logs retrieved from the Kafka Queue whose corresponding service is already registered. The services are stored in the database of the Issue Creator.
+The combined frontend also offers the feature to register multiple services that are to be monitored. Upon registration, the Issue Creator will only assess those logs retrieved from the Kafka Queue whose corresponding service is already registered. Registered services are stored in its own database which can be found in the Issue Creator repository.
```

Comment:
```suggestion
The combined frontend also offers the feature to register multiple services that are to be monitored. Upon registration, the Issue Creator will only assess those logs retrieved from the Kafka Queue whose corresponding service is already registered. Registered services are stored in their own databases which can be found in the Issue Creator repository.
```
falls du sagen wolltest, dass jeder service in einer eigenen datenbank gespeichert wird. oder anders gefragt: in wessen datenbank werden die services gespeichert?